### DISCUSSION TITLE
Problem in jet fire simulation

### DISCUSSION THREAD

**Discussion by DOGzym** [firemodels/fds#13078] [Q&A]
Hi all,
Simulation of jet fire using FDS showed that the temperature and velocity data could not match the experimental phenomena. When the jet velocity is higher, the lift-off phenomenon is shown in the experiment (i.e., there is no combustion reaction in the area closer to the jet exit, and the flame is present only downstream). The lift-off distance is close to 10 cm (determined by the exit velocity and diameter). LES, VLES and DNS have been deployed as the simulation mode. In LES and VLES modes, a high temperature region can be seen at the exit (i.e., there is no lift-off phenomena) regardless of the exit velocity. Is the problem with the combustion reaction or something else? 
The code is attached below.
[JETFIREMODEL.txt](https://github.com/user-attachments/files/15898193/JETFIREMODEL.txt)

**Comment by rmcdermo** [comment]
FDS is not designed to model jet lift off height.  By default, FDS uses the eddy dissipation model (infinitely fast chemistry, mixed is burnt), this is not what you need to capture lift off height; you need to do finite-rate chemistry, and then you are in research territory with FDS.  It is doable, but not by just playing with input parameters.  This is something that is on the cutting edge of our development efforts.  We don't have a simple solution for you.
